src/Export/Adapters/AttributeAdapter.php

```diff
@@ -149,68 +150,24 @@ protected function getManufacturerAttributes(ProductEntity $product): array
     }
 
     /**
+     * @deprecated tag:6.0.0 - Logic was moved to the OptionsAdapter
      * @return Attribute[]
      */
-    protected function getPropertyAttributes(ProductEntity $product): array
+    public function getOptionAttributes(ProductEntity $product): array
```

Comment:
Why was this changed to public now? This will break all the extensions using it and should not be needed.